formatters/svg/svg.go

```diff
@@ -2,16 +2,57 @@
+
 // Option sets an option of the SVG formatter.
 type Option func(f *Formatter)
 
+// FontFamily sets the font-family.
+func FontFamily(fontFamily string) Option { return func(f *Formatter) { f.fontFamily = fontFamily } }
+
+// EmbedFontFile embeds given font file
+func EmbedFontFile(fontFamily string, fileName string) (option Option, err error) {
+	var format FontFormat
+	switch path.Ext(fileName) {
+	case ".woff":
+		format = WOFF
+	case ".woff2":
+		format = WOFF2
+	case ".ttf":
+		format = TRUETYPE
+	default:
+		return nil, errors.New("unexpected font file suffix")
+	}
+
+	if f, err := os.Open(fileName); err == nil {
+		defer f.Close()
+		reader := bufio.NewReader(f)
+		if content, err := ioutil.ReadAll(reader); err == nil {
```

Comment:
You can replace this all with `ioutil.ReadFile()`.

Reply:
oh, nice